.github/workflows/release.yml

```diff
@@ -53,5 +81,14 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          AC_USERNAME: ${{ secrets.AC_USERNAME }}
+          AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
           # Your GoReleaser Pro key, if you are using the 'goreleaser-pro' distribution
           # GORELEASER_KEY: ${{ secrets.GORELEASER_KEY }}
+      -
+        name: Attach produced packages to Github Action
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist/*.tar.gz
```

Comment:
ATM, this creates a single `dist` archive. Is there a way how to upload individual platform-specific archives as standalone items?

Reply:
`actions/upload-artifact` currently can only produce one artifact per step. In the future there might be UI for letting you download specific files from that artifact: https://github.com/actions/upload-artifact/pull/205#issuecomment-841256465. For now I think we can either wait for this improvement, or manually split this step into one per OS, or even one per OS x architecture. What do you think?

Reply:
I'd say we have more important things to work. A single `dist` archive is good enough for now, considering the options available.